info/guide/diktat-coding-convention.md

```diff
@@ -2095,4 +2123,71 @@ interface I {
 object O: I {
     override fun foo() {}
 }
+```
+# <a name="c8"></a> 8. Things that will be moved to a main guide later
```

Comment:
this should not be there yet, bad idea to generate the version of a codestyle

Reply:
OK, reverted